content/blog/adoptopenjdk-welcomes-dragonwell/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: AdoptOpenJDK Welcomes Dragonwell
+date: "2020-11-10T12:00:00+00:00"
+author: andreasahlenstorf
+description: With Alibaba Dragonwell, another flavour of OpenJDK is being built at and distributed by AdoptOpenJDK. Dragonwell is a TCK-certified implementation of Java for large-scale application deployments on Alibaba Cloud and elsewhere.
+tags:
+  - announcement
+---
+
+AdoptOpenJDK is happy to announce that Alibaba decided to join forces with us by building and distributing their [Dragonwell JDK](http://dragonwell-jdk.io/) on our infrastructure. For our users, this means more choice, mainly because Dragonwell offers some exciting features that are unique to Dragonwell. One example is [Wisp](https://github.com/alibaba/dragonwell8/wiki/Wisp-Documentation) that implements coroutines on the JVM.
+
+Over the next weeks, the first releases of Dragonwell 8 and 11 are going to appear on AdoptOpenJDK's website alongside our existing binary offerings. Dragonwell can be consumed like any other binary release by AdoptOpenJDK. The Dragonwell team will also introduce Dragonwell JDK and its unique features in a series of blog posts. Stay tuned!
+
+Dragonwell is a flavour of OpenJDK (like AdoptOpenJDK!). It is a TCK-certified [[1](#fn-tck)<a name="fn-tck-ret"></a>] implementation of Java for large-scale application deployments on Alibaba Cloud and elsewhere. As such, it is a drop-in replacement for any other OpenJDK variant and compatible with all the tools you know, like JDK Flight Recorder and JDK Mission Control.
```

Comment:
Dragonwell's relationship with openjdk:

Alibaba Dragonwell is certified as compatible with the Java SE standard. 
Alibaba Dragonwell is a "friendly fork" under the same licensing terms as the upstream OpenJDK project. Alibaba is committed to collaborating closely with the OpenJDK community and intends to bring as many customized features as possible from Alibaba Dragonwell to the upstream.

Reply:
Included with slight changes.